tools/webhook-apicoverage/resourcetree/README.md

```diff
@@ -47,4 +47,22 @@ Type analysis are defined inside [typeanalyzer_tests](buildChildNodes_test.go)
 A Resource tree is updated using reflect.Value Each node type is expected
 to implement NodeInterface method *updateCoverage(v reflect.Value)*.
 Inisde this method each node updates its nodeData.covered field based on
-whether the reflect.Value parameter being passed is set or not.
```

Comment:
the section of "## Resource Forest" has duplicated content about "an example of ConnectedNodes ...", you may want to correct it in this PR

Reply:
That sneaked in with a bad merge, I have fixed it.